docs/formats.md

```diff
@@ -31,10 +31,57 @@ You use formats in your config file under platforms > [platform] > files > [file
 
 There is an extensive (but not exhaustive) list of [included formats](#pre-defined-formats) available in Style Dictionary.
 
+### Format configuration
```

Comment:
I was just to comment and say this is a great new feature, then looked at my code and fount I am already doing this. 🙈